helpers/contains.js

```diff
@@ -10,7 +10,7 @@
 const factory = () => {
     return function(container, value) {
         const options = arguments[arguments.length - 1];
-        const contained = container.includes(value);
+        const contained = containerIncludesValue(container, value);
```

Comment:
What is the general reason to support more data types?

Reply:
I wanted to make sure to preserve as much backward compatibility as possible. Lodash supported more options so its not as simple of a refactor as I initially thought, we also didnt explicitly test this case, but our public helper docs point out object as a possible value.